packages/patternfly-4/react-core/src/components/ContextSelector/ContextSelectorToggle.tsx

```diff
@@ -1,48 +1,47 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
 import { CaretDownIcon } from '@patternfly/react-icons';
 import styles from '@patternfly/patternfly/components/ContextSelector/context-selector.css';
```

Comment:
Change to `@patternfly/react-styles/components/ContextSelector/context-selector`